gui/wxpython/photo2image/ip2i_manager.py

```diff
@@ -1062,25 +1062,24 @@ def CheckGCPcount(self, msg=False):
         are active for the selected transformation order
         """
         if (
-            (self.GCPcount < 3 and self.gr_order == 1)
-            or (self.GCPcount < 6 and self.gr_order == 2)
-            or (self.GCPcount < 10 and self.gr_order == 3)
+            (self.GCPcount >= 3 or self.gr_order != 1)
+            and (self.GCPcount >= 6 or self.gr_order != 2)
+            and (self.GCPcount >= 10 or self.gr_order != 3)
```

Comment:
The same comment here.

Reply:
see https://github.com/OSGeo/grass/pull/4916#discussion_r1938127084